projects/kit/components/combo-box/combo-box.component.ts

```diff
@@ -143,14 +143,8 @@ export class TuiComboBoxComponent<T>
     }
 
     @tuiPure
-    computeContext(
-        $implicit: T | null,
-        active: boolean,
-    ): TuiValueContentContext<T | null> {
-        return {
-            $implicit,
-            active,
-        };
+    computeContext($implicit: T, active: boolean): TuiValueContentContext<T> {
```

Comment:
It's probably unnecessary and can be inlined in the template

Reply:
fixed